packages/slice-machine/pages/index.tsx

```diff
@@ -165,15 +155,45 @@ const CustomTypes = () => {
         }
         breadrumbHref="/"
       />
-      {!customTypes.length ? <Emptystate /> : null}
-      <Grid
-        elems={customTypes}
-        renderElem={(ct: CtPayload) => (
-          <Link passHref href={`/cts/${ct.id}`} key={ct.id}>
-            <Card ct={ct} />
-          </Link>
-        )}
-      />
+      {!customTypes.length ? (
+        <Box
```

Comment:
It's not mandatory but personnaly and to improve the. visibility of the file, I would externalise the empty state component, maybe in a file.